### ISSUE TITLE
Init fails with relative path.

### ISSUE THREAD

**Issue by mapk0y** [opencontainers/runc#12]
```console
host:~/runcdir# ls -l ../runc
-rwxr-xr-x 1 root root 7467584  6月 23 06:30 ../runc

host:~/runcdir# ls -l
total 8
-rw-r--r--  1 root root 1457  6月 23 06:31 container.json
drwxr-xr-x 21 root root 4096  6月 23 05:06 rootfs

host:~/runcdir# ../runc
Timestamp: 2015-06-23 06:31:13.157456619 +0900 JST
Code: System error

Message: fork/exec ../runc: no such file or directory

Frames:
---
0: start
Package: github.com/opencontainers/runc/libcontainer.(*initProcess)
File: process_linux.go@183
---
1: Start
Package: github.com/opencontainers/runc/libcontainer.(*linuxContainer)
File: container_linux.go@107
---
2: execContainer
Package: main
File: run.go@41
---
3: func·006
Package: main
File: main.go@88

4: Run
Package: github.com/codegangsta/cli.(*App)
File: app.go@159
---
5: main
Package: main
File: main.go@96
---
6: main
Package: runtime
File: proc.go@63
---
7: goexit
Package: runtime
File: asm_amd64.s@2232
```

absolute path OK
```console
host:~/runcdir# /root/runc
$ 
host:~/runcdir# ~/runc
$ 
```

Because `../runc` is not exist  `../runc` when `chdir` to rootfs.
I think this is irregular.
But I want you to fix it.

**Comment by ctrlrsf** [comment]
The error message is a bit verbose, but nonetheless it explains the problem 3 lines in.

```
host:~/runcdir# ../runc
Timestamp: 2015-06-23 06:31:13.157456619 +0900 JST
Code: System error

Message: fork/exec ../runc: no such file or directory
...
```

**Comment by estesp** [comment]
Fixed by #28

**Comment by mapk0y** [comment]
Thanks @estesp .

I have confirmed that there is no problem.

```console
host:~/runcdir# ../runc.old
Timestamp: 2015-06-25 09:20:35.869991947 +0000 GMT
Code: System error

Message: fork/exec ../runc.old: no such file or directory
(snip)

host:~/runcdir# ../runc.new
root@shell:/#
```
